tests/test0185.d

```diff
@@ -0,0 +1,22 @@
+//T compiles:yes
+//T retval:0
+//T has-passed:yes
+// Tests nested switch 
+int main() {
```

Comment:
@UplinkCoder for this test case it was failing to compile with error fall through is not enable at line case 2: . Should we modify the test case ??